modules/@apostrophecms/i18n/index.js

```diff
@@ -78,7 +78,14 @@ module.exports = {
       // not set already by explicit query parameters. Conversely,
       // if the parameters were set, rewrite `_id` accordingly.
       // Returns `_id`, after rewriting if appropriate.
-      inferIdLocaleAndMode(req, _id) {
+      //
+      // If `localize` is `true`, or the name of a type that is localized,
+      // the method has its effect as described above. Otherwise it has no effect
+      // and returns `_id` unmodified.
+      inferIdLocaleAndMode(req, _id, localize) {
+        if ((localize === false) || ((localize !== true) && (!self.apos.doc.getManager(localize).isLocalized()))) {
```

Comment:
it seems weird to have an argument that is, in part at least, saying whether or not this method should have been run in the first place.